src/js/dom.js

```diff
@@ -0,0 +1,59 @@
+
+export function appendToMain($element) {
+    const $main = document.getElementById('app');
+    $main.appendChild($element);
+    
+}
+
+function hidePokemonUI() {
+    document.getElementById('pokemon-info').remove();
+
+}
+
+function showList() {
```

Comment:
What's `List`? I think this function should be more descriptive, maybe `showPokemons`?

Reply:
I will try to be less general in specific contexts